planner/cascades/testdata/stringer_suite_out.json

```diff
@@ -0,0 +1,177 @@
+[
+  {
+    "Name": "TestGroupStringer",
+    "Cases": [
+      {
+        "SQL": "select b from t where a > 1 and b < 1",
+        "Result": [
+          "Group#0 Column: [Column#13] Unique key: []",
+          "    Projection_3 input:Group#1, Column#2",
```

Comment:
What about `input:[Group#1], Column#2`?

Reply:
I'm ok to do so. What do you think? @zz-jason

Reply:
addressed